dumper/lib-ext-hive-metastore/build.gradle

```diff
@@ -30,6 +30,7 @@ dependencies {
     api libs.libthrift
     implementation libs.guava
     implementation libs.jackson.databind
+    compileOnly libs.javax.annotation.api
```

Comment:
Isn't `compileOnly` the "old" way of declaring a compile-time dependency, shouldn't this be `implementation` instead?

Reply:
The [docs](https://docs.gradle.org/current/userguide/java_plugin.html#dependency_declaration_configurations) say `implementation` is compile-time and runtime, `compileOnly` is only compile-time. I believe these annotations only need to be on the classpath at compile-time.

Reply:
Ah, got it, never mind, I was mixing up `compileOnly` with `compile`, the latter of which is the old/deprecated way; https://docs.gradle.org/6.9.4/userguide/java_plugin.html#sec:java_plugin_and_dependency_management